mobile/src/main/res/values-de/strings.xml

```diff
@@ -1,139 +1,138 @@
-    <string name="error_certificate_expired">SSL-Zertifikat abgelaufen. Bitte erstellen Sie ein neues Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung</string>
-    <string name="error_certificate_not_valid_yet">SSL-Zertifikat noch nicht gültig. Bitte erstellen Sie ein gültiges Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung. Überprüfen Sie auch die Uhrzeit auf Ihrem Smartphone</string>
-    <string name="error_certificate_revoked">SSL-Zertifikat widerrufen. Bitte erstellen Sie ein neues Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung</string>
-    <string name="error_http_code_401">Authentifizierung fehlgeschlagen. Bitte überprüfen Sie Nutzername und Passwort bzw. Client-Zertifikat (HTTP-Statuscode 401)</string>
-    <string name="error_http_code_403">Authentifizierung fehlgeschlagen. Bitte überprüfen Sie Nutzername und Passwort bzw. Client-Zertifikat (HTTP-Statuscode 403)</string>
-    <string name="error_http_code_407">Authentifizierung fehlgeschlagen. Bitte überprüfen Sie Nutzername und Passwort bzw. Client-Zertifikat (HTTP-Statuscode 407)</string>
-    <string name="error_http_code_414">openHAB-URL zu lang (HTTP-Statuscode 414)</string>
-    <string name="error_http_code_426">HTTPS wird gefordert. Bitte ändern Sie die openHAB-URL (HTTP-Statuscode 426)</string>
-    <string name="error_http_code_495">Ungültiges Client-Zertifikat (HTTP-Statuscode 495)</string>
-    <string name="error_http_code_496">Client-Zertifikat wird gefordert (HTTP-Statuscode 496)</string>
-    <string name="error_http_code_502">Ihr Reverse-Proxy bekam eine ungültige Antwort von openHAB (HTTP-Statuscode 502)</string>
-    <string name="error_http_code_503">openHAB steht temporär nicht zur Verfügung (HTTP-Statuscode 503)</string>
-    <string name="error_http_code_504">Gateway Time-out. Ihr Reverse-Proxy läuft, aber bekam keine Antwort von openHAB (HTTP-Statuscode 504)</string>
-    <string name="error_http_code_507">openHAB hat nicht genug freien Speicher (HTTP-Statuscode 507)</string>
-    <string name="error_http_code_511">Netzwerkauthentifizierung wird gefordert (HTTP-Statuscode 511)</string>
-    <string name="error_network_type_unsupported">Netzwerktyp (%s) wird nicht unterstützt</string>
-    <string name="unknown">Unbekannt</string>
-    <string name="settings_debug_messages_summary">Einige Meldungen enthalten mehr Informationen</string>
-    <string name="settings_debug_messages_title">Debug-Meldungen anzeigen</string>
-    <string name="error_couldnt_determine_openhab_url">Konnte openHAB-URL nicht ermitteln</string>
-    <string name="error_unable_to_resolve_hostname">Fehler beim Auflösen des Hostnamens</string>
-    <string name="set">Setzen</string>
-    <string name="cancel">Abbrechen</string>
-    <string name="dlg_notification_title">Benachrichtigung</string>
+  <!-- General app strings -->
```

Comment:
we should probably format these strings before the import of strings from crowdin, as otherwise checking the diff is impossible :)

Reply:
Hey Florian, did you see my remarks in the OP?

Reply:
it seems I haven't, sorry! :/

Reply:
No problem. That's a hidden feature by GitHub